BookingsApi/BookingsApi/Controllers/WorkAllocationsController.cs

```diff
@@ -146,6 +152,13 @@ public async Task<IActionResult> AllocateHearingManually([FromBody] UpdateHearin
                 var list = await _hearingAllocationService.AllocateHearingsToCso(postRequest.Hearings, postRequest.CsoId);
                 
                 var dtos = _hearingAllocationService.CheckForAllocationClashes(list);
+                
+                // need to broadcast acknowledgment message for the allocation
+                if (list.Count > 0)
```

Comment:
What happens if one of the allocations fail? If one fails then no notifications are sent

Reply:
if one allocation is failing then all the allocation are failing. Therefore no message will be sent